packages/winston/src/index.ts

```
@@ -0,0 +1,71 @@
+  Logger as MeshLogger,
+} from '@graphql-mesh/types';
+import type { Logger as WinstonLogger } from 'winston';
+
+function prepareArgs(lazyArgs: LazyLoggerMessage[]) {
+  const flattenedArgs = lazyArgs
+    .flatMap((lazyArg) => (typeof lazyArg === 'function' ? lazyArg() : lazyArg))
+    .flat(Infinity)
+    .map((arg) => {
+      if (typeof arg === 'string') {
+        try {
+          arg = JSON.parse(arg);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)